uint16 planeID

bool need_idel

bool need_take_off

bool need_land

bool need_formation

bool need_protected
